src/services/services.ts

```diff
@@ -504,8 +504,9 @@ function createChildren(node: Node, sourceFile: SourceFileLike | undefined): rea
         });
         return children;
     }
-
+    const languageVariant = (sourceFile as SourceFile)?.languageVariant ?? LanguageVariant.Standard;

```

Comment:
I would personally just declare `languageVariant` as optional in `SourceFileLike`.